drivers/include/xbee.h

```diff
@@ -104,6 +117,7 @@ typedef struct {
     gpio_t sleep_pin;                   /**< GPIO pin connected to SLEEP */
     ng_nettype_t proto;                 /**< protocol the interface speaks */
     uint8_t options;                    /**< options field */
+    uint8_t addr_flags;                 /**< address flags as defined above */
```

Comment:
I wasn't sure if these were not device dependent, since they are added to the tx-buffer in https://github.com/authmillenon/RIOT/blob/xbee/enh/addr-len/drivers/xbee/xbee.c#L553